packages/flame_forge2d/example/lib/balls.dart

```diff
@@ -60,39 +60,63 @@ class Ball extends BodyComponent {
       }
     }
   }
+
+  @override
+  void beginContact(Contact contact) {
+    final otherObject = contact.bodyA.userData == this
```

Comment:
The user shouldn't have to do this boilerplate (since they didn't before)

Reply:
I will look into this. I need to double-check the logic that places one body over another. That might simplify this.

However, a `Body` can have multiple `Fixtures`. `Fixtures` can have `userData` too. So if a `Body` contacts with another `Body` that has a defined `Fixture` userData and a `Body` defined userData what would be the `other`?